tests/worker.test.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, it, expect, beforeAll, afterAll } from 'vitest';
+import { AgentRunner } from '../worker/runner';
+import { AGENT_SERVER_PORT } from '../server/constants';
+
+describe('Worker Health Check', () => {
```

Comment:
Update our ci GitHub action such that we run the action once tests change and that we run these tests